janusgraph-server/src/main/java/org/janusgraph/graphdb/server/JanusGraphServer.java

```diff
@@ -17,52 +17,86 @@
+            gremlinSettings = GremlinSettingsUtils.configureDefaultSerializersIfNotSet(Settings.read(confPath));
+            gremlinServer = new GremlinServer(gremlinSettings);
+            serverStarted = CompletableFuture.allOf(gremlinServer.start());
+        }
+        catch(Exception ex){
+            serverStarted.completeExceptionally(ex);
+        }
+        return serverStarted;
+    }
+
+    public synchronized CompletableFuture<Void> stop() {
+        if (gremlinServer == null) {
+            return CompletableFuture.completedFuture(null);
+        }
+        if (serverStopped != null) {
+            return serverStopped;
+        }
+        serverStopped = gremlinServer.stop();
+        return serverStopped;
+    }
+
     private static String getHeader() {
         return "                                                                      " + System.lineSeparator() +
-        "   mmm                                mmm                       #     " + System.lineSeparator() +
-        "     #   mmm   m mm   m   m   mmm   m\"   \"  m mm   mmm   mmmm   # mm  " + System.lineSeparator() +
-        "     #  \"   #  #\"  #  #   #  #   \"  #   mm  #\"  \" \"   #  #\" \"#  #\"  # " + System.lineSeparator() +
-        "     #  m\"\"\"#  #   #  #   #   \"\"\"m  #    #  #     m\"\"\"#  #   #  #   # " + System.lineSeparator() +
-        " \"mmm\"  \"mm\"#  #   #  \"mm\"#  \"mmm\"   \"mmm\"  #     \"mm\"#  ##m#\"  #   # " + System.lineSeparator() +
-        "                                                         #            " + System.lineSeparator() +
-        "                                                         \"            " + System.lineSeparator();
+            "   mmm                                mmm                       #     " + System.lineSeparator() +
+            "     #   mmm   m mm   m   m   mmm   m\"   \"  m mm   mmm   mmmm   # mm  " + System.lineSeparator() +
+            "     #  \"   #  #\"  #  #   #  #   \"  #   mm  #\"  \" \"   #  #\" \"#  #\"  # " + System.lineSeparator() +
+            "     #  m\"\"\"#  #   #  #   #   \"\"\"m  #    #  #     m\"\"\"#  #   #  #   # " + System.lineSeparator() +
+            " \"mmm\"  \"mm\"#  #   #  \"mm\"#  \"mmm\"   \"mmm\"  #     \"mm\"#  ##m#\"  #   # " + System.lineSeparator() +
+            "                                                         #            " + System.lineSeparator() +
+            "                                                         \"            " + System.lineSeparator();
     }
 
     private static void printHeader() {
         logger.info(getHeader());
-        logger.info("JanusGraph Version: " +  Manifests.read("janusgraphVersion"));
-        logger.info("TinkerPop Version: " +  Manifests.read("tinkerpopVersion"));
+        logger.info("JanusGraph Version: " + Manifests.read("janusgraphVersion"));
+        logger.info("TinkerPop Version: " + Manifests.read("tinkerpopVersion"));
```

Comment:
Since you are formatting this line, do you want to change it to slf4j style, i.e. using "{}" rather than string concatenation?